gui/wxpython/mapwin/buffered.py

```diff
@@ -1615,7 +1615,7 @@ def OnButtonDClick(self, event):
                 screenCoords[0],
                 screenCoords[1],
                 self.hitradius)
-            if idlist:
+            if len(idlist) > 1:
```

Comment:
@wenzeslaus @petrasovaa Do we need to deal with id 99 explicitly also here?

Reply:
I think so, the suggested fix won't work when you have only 1 overlay and no map. So something like this should work better:

             if idlist and idlist[0] != 99:

Reply:
Yes, of course, I updated it. Thanks for the correction.